.gitignore

```diff
@@ -0,0 +1,2 @@
+# dependencies
+/node_modules
```

Comment:
- [ ] kindly consider including your test.md file here to meet the below requirement 👇 
> There is a .gitignore file listing `test.md` file and `node_modules` directory (as it can be added by accident while setting up Linters).

Reply:
Hello @divinecharlotte  ,
I am confused about how to reach out to you without making another pull request after I fix my code. And will it cost me an extra review? Please help me out here.